src/logger.ts

```diff
@@ -315,10 +330,31 @@ export class Logger {
       throw new SfdxError('RedundantRootLogger');
     }
 
+    // Inspect format to know what logging format to use then delete from options to
+    // ensure it doesn't conflict with Bunyan.
+    this.format = options.format || LoggerFormat.JSON;
+    delete options.format;
+
+    // If the log format is LOGFMT, we need to convert any stream(s) into a LOGFMT type stream.
+    if (this.format === LoggerFormat.LOGFMT && options.stream) {
+      const ls: LoggerStream = this.createLogFmtFormatterStream({ stream: options.stream });
+      options.stream = ls.stream;
+    }
+    if (this.format === LoggerFormat.LOGFMT && options.streams) {
+      const logFmtConvertedStreams: LoggerStream[] = [];
+      options.streams.forEach((ls: LoggerStream) => {
+        logFmtConvertedStreams.push(this.createLogFmtFormatterStream(ls));
+        options.streams = logFmtConvertedStreams;
```

Comment:
Seems like this should be out of the loop.

Reply:
Good catch!  Fixed.